plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -1217,6 +1218,27 @@ def update_auth_policy_server(self, ipAddress):
             if is_ise_server:
                 trusted_server = self.want.get("trusted_server")
                 self.accept_cisco_ise_server_certificate(ipAddress, trusted_server)
+                response = self.dnac._exec(
+                    family="system_settings",
+                    function='get_authentication_and_policy_servers',
+                    params={"is_ise_enabled": True}
+                )
+                response = response.get("response")
+                if response is None:
+                    self.msg = "Failed to retrieve the information from the API 'get_authentication_and_policy_servers' of {0}." \
+                               .format(ipAddress)
+                    self.status = "failed"
+                    return
+
+                ise_server_details = get_dict_result(response, "ipAddress", ipAddress)
+                ise_state_list = {"FAILED", "INPROGRESS"}
```

Comment:
ise_state_set instead of ise_state_list 
or ise_states

ise_state = ise_server_details.get("state")

if ise_state in ise_states: 
or
if ise_state in ise_state_set: